pkg/util/collections/includes_excludes.go

```diff
@@ -154,6 +155,48 @@ func ValidateIncludesExcludes(includesList, excludesList []string) []error {
 	return errs
 }
 
+// ValidateNamespaceIncludesExcludes checks provided lists of included and
+// excluded namespaces to ensure they are a valid set of IncludesExcludes data.
+func ValidateNamespaceIncludesExcludes(includesList, excludesList []string) []error {
+	errs := ValidateIncludesExcludes(includesList, excludesList)
+
+	includes := sets.NewString(includesList...)
+	excludes := sets.NewString(excludesList...)
+
+	for _, itm := range includes.List() {
+		// Although asterisks is not a valid Kubernetes namespace name, it is
+		// allowed here.
+		if itm != "*" {
+			if nsErrs := validateNamespaceName(itm); nsErrs != nil {
+				errs = append(errs, nsErrs...)
+			}
+		}
+	}
+
+	for _, itm := range excludes.List() {
+		// Asterisks in excludes list have been checked previously.
+		if itm != "*" {
+			if nsErrs := validateNamespaceName(itm); nsErrs != nil {
+				errs = append(errs, nsErrs...)
+			}
+		}
+	}
+
+	return errs
+}
+
+func validateNamespaceName(ns string) []error {
+	var errs []error
+
+	if errMsgs := validation.ValidateNamespaceName(ns, false); errMsgs != nil {
+		for _, msg := range errMsgs {
+			errs = append(errs, errors.Errorf("namespace, %q: %s", ns, msg))
```

Comment:
When this is included with other validation errors, it's a little difficult to parse. For example:
```
$ velero backup create b1 --include-namespaces=foo --exclude-namespaces=foo,invalid/ns
An error occurred: [excludes list cannot contain an item in the includes list: foo, namespace, "invalid/ns": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')]
```

To help with that, we could remove the comma and use the following format string: `invalid namespace %q: %s`.

Reply:
No problem--format string has been updated.